pyproximal/proximal/DykstraLikeProximal.py

```diff
@@ -0,0 +1,297 @@
+from typing import List, Any
+
+from pylops.utils.typing import NDArray
+from pylops.utils.backend import get_array_module
+
+from pyproximal.ProxOperator import ProxOperator, _check_tau
+
+
+class DykstraLikeProximal(ProxOperator):
+    r"""Proximal operator of a sum of two or more convex functions
+    using Dykstra-like algorithm.
+
+
+    Parameters
+    ----------
+    ops : :obj:`List[ProxOperator]`
+        A list of proximable functions :math:`f_1, \ldots, f_m`.
+    weights : :obj:`np.ndarray` or :obj:`List[float]` or :obj:`None`, optional, default=None
+        Weights :math:`\sum_{i=1}^m w_i = 1, \ 0 < w_i < 1`,
+        used when :math:`m > 2`, or :math:`m = 2` and `use_parallel=True`.
+        Defaults to None, which means :math:`w_1 = \cdots = w_m = \frac{1}{m}.`
+    max_iter : :obj:`int`, optional, default=1000
+        The maximum number of iterations.
+    tol : :obj:`float`, optional, default=1e-7
+        Torrelance to stop the iteration.
```

Comment:
Torrelance -> Tolerance

Also I would be more descriptive... for example in `ProximalPoint` we have the following description for tolerance: `Tolerance on change of objective function (used as stopping criterion). If ``tol=None``, run until ``niter`` is reached`